### ISSUE TITLE
errors trying to build assignment 2 with Cmake

### ISSUE THREAD

**Issue by NickMakesGames** [cisc3620/docs#3]
I ran the command 
  >cmake -G "Visual Studio 14" ..

in the build subdirectory of the second assignment's code, and I encountered this error. Is this an issue with the repository or my personal setup?

CMake Error at CMakeLists.txt:7 (add_subdirectory):
The source directory

C:/Users/Nicho/Desktop/ComputerGraphics/2-draw-house-NickMakesGames/Glitter/Vendor/glfw

does not contain a CMakeLists.txt file.

**Comment by TS96** [comment]
I think this is because when you cloned the repository, you forgot the "--recursive" option so you're missing  files.

**Comment by rivlev** [comment]
Please close if this fixes your issue

**Comment by NickMakesGames** [comment]
I haven't had a chance to try that until just now. I deleted my local files and re-cloned the repository, making sure to include the --recursive flag, but I ran into the same issue. I checked the actual gitHub page for my project, hosted here:
 https://github.com/cisc3620/2-draw-house-NickMakesGames
and it turns out that there's actually just nothing in the build sub-directory there. Is anybody else encountering that issue?
